src/main/java/net/minecraft/launchwrapper/LogWrapper.java

```diff
@@ -55,7 +55,7 @@ public static void log(Level level, Throwable ex, String format, Object... data)
         if (format.contains("{}")) {
             log.myLog.log(level, format, data);
```

Comment:
You can pass in exceptions to this format of log calls too IIRC, the throwable needs to be appended to the data array